packages/core/admin/admin/src/pages/Webhooks/ListView/index.js

```diff
@@ -267,6 +200,36 @@ function ListView() {
     }
   };
 
+  const handleSelectAllCheckbox = () => {
```

Comment:
In that function, you loop over webhooks 2 times. 
The first time to extract ids. And the second time to select each webhook of the webhookIds.
I think you don't need to do that. You don't need to extract the ids of webhooks. Since you can access the state in the reducer function, you only need to create a function that select all the webhooks.

```js
// Webhooks/ListView/index.js
const handleSelectAllCheckbox = () => {
     dispatch({
      type: 'SET_ALL_WEBHOOKS_TO_DELETE',
    });
};
```

```js
// Webhooks/ListView/reducer.js
...
case 'SET_ALL_WEBHOOKS_TO_DELETE': {
  const { webhooks, webhooksToDelete } = state;

  if (webhooks.length === webhooksToDelete.length) {
    draftState.webhooksToDelete = initialState.webhooksToDelete;
  } else {
    draftState.webhooksToDelete = webhooks.map(webhook => webhook.id);
  }
  break;
}
...
```

Reply:
Don't forget to add some test to this reducer case 😄